### PR TITLE
Add aws-workers-rhel8-ovn jobs

### PR DESCRIPTION
**Pull request by jluhrsen**
jobs like this should help to prevent release blockers like
we got here: https://bugzilla.redhat.com/show_bug.cgi?id=2052393

Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml [hunk 1]
```diff
@@ -806,6 +806,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-workers-rhel8,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (kikisdeliveryservice) [inline ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml:810]: why is this true in 4.9?

**Review by kikisdeliveryservice — COMMENTED** [review]
one comment

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml (hunk 1 above)
Comment (jluhrsen) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml:810]: by default it's true and looks like it's that way from 4.9 and lower. here is the [commit](https://github.com/openshift/release/commit/a2db9bce045) that changed it to false.

I guess the idea is that we can skip these kinds of jobs if it's a change to something unrelated to actual functional code.
I'm not sure why it isn't there for 4.9 and lower though. if we want to do that, I can do it in a different PR.

Comment (kikisdeliveryservice) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml:810]: Can you please update this to `always_run: false` here to match the rest of the PR.

Comment (jluhrsen) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml]: ok, added a new commit to this PR to take care of that.

**Comment by jluhrsen** [comment]
/hold
need to get this right. the first try was not deploying with ovnk like I wanted.

**Comment by jluhrsen** [comment]
/hold cancel

@kikisdeliveryservice , how does this look now?

**Comment by kikisdeliveryservice** [comment]
> /hold cancel
> 
> @kikisdeliveryservice , how does this look now?

This looks good to me!!

/approve

**Comment by kikisdeliveryservice** [comment]
For final reqd approval:

/assign  @jacobtanenbaum @trozet

**Comment by trozet** [comment]
@jluhrsen can you change the owners from Alex to Jaime? Why are the rehearse jobs failing?

**Comment by jluhrsen** [comment]
> @jluhrsen can you change the owners from Alex to Jaime? Why are the rehearse jobs failing?

@trozet 

ok, owners files are updated.

I think the 4.12 rehearsal job is failing because it's 4.12 and that doesn't mean anything yet (???). I only added it because there was already a workers-rhel (openshift-sdn) job in that file.

the 4.10 job failed because of e2e test failures. did not look in to them. but since it wasn't a failure from job config I think we can move forward with this and if they are consistent we can dig in them and get them resolved with new bzs

**Comment by kikisdeliveryservice** [comment]
readding my
/lgtm

**Comment by jluhrsen** [comment]
had to rebase...

**Comment by kikisdeliveryservice** [comment]
oh noo let's try again

/approve
/lgtm

File: ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
+			"dcbw",
+			"knobunc",
+			"squeed",
+			"trozet"
 		],
 		"reviewers": [
-			"aravindhp",
-			"sebsoto",
-			"JacobTanenbaum"
+			"alexanderconstantinescu",
```
Comment (kikisdeliveryservice) [inline ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json:12]: This one is causing some issues I think..

**Comment by jluhrsen** [comment]
@aravindhp, @JacobTanenbaum, @sebsoto, could someone PTAL at this and hopefully give /approve?

**Pull request merged**